blocks/editable/readme.md

```diff
@@ -0,0 +1,125 @@
+# `Editable`
```

Comment:
Minor: This should probably have been named uppercase `README.md` for consistency with other documents.

Reply:
Yep, did this in #4779